### DISCUSSION TITLE
How to show react bootstrap modal when the webgl canvas is in fullscreen mode

### DISCUSSION THREAD

**Discussion by liew312** [jeffreylanters/react-unity-webgl#227] [Help Wanted!] [answered]
Hi, the modal is not showing up when the webgl canvas is in fullscreen mode. What can I do so that the modal will be shown? Thank you.

**Comment by jeffreylanters** [comment, marked as answer]
Hi. I don't think this is possible at all, when entering full-screen mode, WebGL runs in a full screen window and nothing can get on top of it. This has nothing to do with this module but is a limitation from the browser. If any kind of pop-up should overlay the game, it should be part of it rather than the page where it's being rendered onto.

**Reply by liew312** [reply]
Thank you.
